boards/adafruit/qt_py_esp32s3/adafruit_qt_py_esp32s3_procpu_psram.yaml

```diff
@@ -0,0 +1,18 @@
+identifier: adafruit_qt_py_esp32s3@psram/esp32s3/procpu
+name: Adafruit QT Py ESP32S3 PSRAM PROCPU
+type: mcu
+arch: xtensa
+toolchain:
+  - zephyr
+supported:
+  - gpio
+  - uart
+  - i2c
+  - spi
+  - can
+  - counter
+  - watchdog
+  - entropy
+  - pwm
+  - dma
```

Comment:
i2s missed here also

Reply:
This board is almost completely copied from seeed_xiao_esp32s3, and its missing from there too. I'll add it to this board, but we should add it to the Xiao board if that truely is the case.